### PR TITLE
Changes defaults in state tool installers

### PR DESCRIPTION
**Pull request by rawktron**
Makes modifications to the install scripts to change the default behaviour to default to YES rather than NO and adds additional contextual information in the prompt.

### CODE REVIEW

File: installers/install.ps1 [hunk 1]
```diff
@@ -63,9 +63,9 @@ function isAdmin
 
 function promptYN([string]$msg)
 {
-    $response = Read-Host -Prompt $msg" [y/N]`n"
+    $response = Read-Host -Prompt $msg" [Y/n]`y"
```
Comment (Naatan) [inline installers/install.ps1:66]:
```suggestion
    $response = Read-Host -Prompt $msg" [Y/n]`n"
```

I think that was a line ending

**Review by Naatan — CHANGES_REQUESTED** [review]

File: installers/install.ps1 (hunk 1 above)
Comment (rawktron) [reply installers/install.ps1:66]: Fixed.

Comment (rawktron) [reply installers/install.ps1:66]: Hold on, that commit isn't showing up here.

Comment (rawktron) [reply installers/install.ps1]: Ok fixed -- weird GitHub issue. I had to push a new commit to get the other one to show up here.

**Review by Naatan — APPROVED** [review]

**Pull request merged**